lib/ae_mdw/db/util.ex

```diff
@@ -30,7 +31,7 @@ defmodule AeMdw.Db.Util do
     do: read_tx(txi) |> one!
 
   def read_block({_, _} = bi) do
-    case Database.fetch(~t[block], bi) do
+    case RocksDbCF.dirty_fetch(~t[block], bi) do
```

Comment:
Direct use of `RocksDbCF` as read through an ongoing transaction is required (different from the way `Database.dirty_read/2` currently works for other tables). So this avoids same name for different behaviors.